test.rb

```diff
@@ -0,0 +1,41 @@
+# Bubble sort
+# frozen_string_literal: true
+
+# rubocop:disable MethodLength
+def bubble_sort(arr)
+  temporal = true
+  i = arr.length
+  while temporal
+    temporal = false
+    (i - 1).times do |x|
+      if arr[x] > arr[x + 1]
+        arr[x], arr[x + 1] = arr[x + 1], arr[x]
+        temporal = true
+      end
+    end
+  end
+  print arr
+end
```

Comment:
* [ ] bubble_sort sorts an array of numbers in increasing order
```ruby
unsorted = (1..10).to_a.reverse!
p bubble_sort(unsorted) # returns nil instead of a sorted array
```

Reply:
We already change the errors, the method is returning a sorted array.